CryptoPay/CryptoPay.csproj

```diff
@@ -19,25 +19,21 @@
         <Copyright>Copyright © CryptoPay.NET 2023</Copyright>
         <PackageProjectUrl>https://github.com/WinoGarcia/CryptoPay</PackageProjectUrl>
         <PackageReleaseNotes></PackageReleaseNotes>
-        <TargetFramework>net6.0</TargetFramework>
+        <TargetFramework>net8.0</TargetFramework>
     </PropertyGroup>
 
     <ItemGroup>
-      <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
+        <None Include="..\package-icon.png">
+            <Pack>True</Pack>
+            <PackagePath></PackagePath>
+            <Link>package-icon.png</Link>
+        </None>
     </ItemGroup>
 
     <ItemGroup>
-      <None Include="..\package-icon.png">
-        <Pack>True</Pack>
-        <PackagePath></PackagePath>
-        <Link>package-icon.png</Link>
-      </None>
-    </ItemGroup>
-
-    <ItemGroup>
-      <Content Include="..\README.md">
-        <Link>README.md</Link>
-      </Content>
+        <Content Include="..\README.md">
+            <Link>README.md</Link>
```

Comment:
When updating the library via nuget, this file will appear in the project every time. Maybe you shouldn’t leave a link to it, otherwise it’s annoying to hide it from your projects every time.